src/main/resources/META-INF/sisu/javax.inject.Named

```diff
@@ -0,0 +1 @@
+org.codehaus.mojo.flatten.DirectDependenciesInheritanceAssembler
```

Comment:
sorry for the stupid question, but if everything is hardcoded against the implementation why do we need @Inject anyways? Because DirectDependenciesInheritanceAssembler needs to be configured and requires injections from the container? If the latter is true, I understand and we can resolve this conversation.

Reply:
Yes, the problem with the Assembler is, that it is contributed from the container